elements/line-status.js

```diff
@@ -3,12 +3,18 @@ const html = Tram.html({
   'line-status-line': require('../elements/line-status-line')
 })
 
-module.exports = (attrs) => {  
+module.exports = (attrs) => {
+  const nav = () => {
```

Comment:
@JRJurman I realized I'm confused why you suggested this. All of the lines are displayed at the same time; there's nothing you can explicitly navigate to right now—is it for futureproofing?